### PR TITLE
feat: add firestore flags to firebase open command

### PR DESCRIPTION
**Pull request by rosariopfernandes**
### Description

Now that Cloud Firestore has its own link on the Firebase Console, it allows for firestore flags on the `firebase open` command.

### Sample Commands

```shell
# Launch Cloud Firestore on the Firebase Console
firebase open firestore

# Launch the "Firestore Rules" tab on the Firebase Console
firebase open firestore:rules

# Launch the "Firestore Indexes" tab on the Firebase Console
firebase open firestore:indexes

# Launch "Firestore Usage" tab on the Firebase Console
firebase open firestore:usage
```

### CODE REVIEW

File: src/commands/open.ts [hunk 1]
```diff
@@ -27,6 +27,10 @@ const LINKS: Link[] = [
   { name: "Database: Rules", arg: "database:rules", consolePath: "/database/rules" },
   { name: "Docs", arg: "docs", url: "https://firebase.google.com/docs" },
   { name: "Dynamic Links", arg: "links", consolePath: "/durablelinks" },
+  { name: "Firestore: Data", arg: "firestore", consolePath: "/firestore/data" },
+  { name: "Firestore: Rules", arg: "firestore:rules", consolePath: "/firestore/rules" },
+  { name: "Firestore: Indexes", arg: "firestore:indexes", consolePath: "/firestore/indexes" },
```
Comment (rosariopfernandes) [inline src/commands/open.ts:32]: Not sure how useful this one would be, considering that we can check the current deployed indexes using the `firestore:indexes` command.

**Comment by samtstern** [comment]
@rosariopfernandes thanks for this, looks cool!

@bkendall do you think this needs an API review?  I personally don't think so since `firestore:rules`, etc are all well-defined slugs used in `deploy` and `open` was already reviewed.

**Comment by mbleigh** [comment]
I think this is fine without an API review.

**Comment by samtstern** [comment]
@rosariopfernandes thank you!

**Pull request merged**